assets/src/userInTestGroup.ts

```diff
@@ -2,6 +2,7 @@ import getTestGroups from "./userTestGroups"
 
 export enum TestGroups {
   DemoMode = "demo-mode",
+  DetourPanel = "detour-panel",
```

Comment:
**question:** I forget, have we decided on an overall test group for this? Should this be that test group or should the dummy page have its own test group? Either way I'm not sure `detour-panel` is really the best name, maybe just `detours` if it's for the overall feature or `dummy-detour-page` if it's just for this page.

Reply:
We did not decide on a name. I just came up with this spur-of-the-moment.

Let's chat tomorrow about what a good name and scope would be for this. I think I've sort of talked myself into liking `dummy-detour-panel` so that it becomes pretty obvious that we should get rid of it once we don't need the dummy panel anymore.

But maybe we want like... `detour-alpha` or something like that so that we can just add ourselves to that group and then forget?

Reply:
> Let's chat tomorrow about what a good name and scope would be for this. I think I've sort of talked myself into liking `dummy-detour-panel` so that it becomes pretty obvious that we should get rid of it once we don't need the dummy panel anymore.

Sounds good!